src/main/java/com/kodilla/ecommercee/domain/Cart.java

```diff
@@ -0,0 +1,15 @@
+package com.kodilla.ecommercee.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+public class Cart {
```

Comment:
Czy ta klasa jest potrzebna na tym etapie?

Reply:
W sumie na tym etapie to nie jest potrzebna klasa Cart, będzie potrzebna podczas mapowania obiektu klasy CartDto na obiekt klasy Cart.

Reply:
Zgadza się, Cart to encja, usuń ją proszę i sprawdź CircleCI